scripts/linters/custom_eslint_checks/lib/rules/break-after-parens.js

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Lint check to ensure that there is a break after parenthesis
+ * in case of multiline hanging indentation.
+ */
+'use strict';
+
+module.exports = {
+  meta: {
+    type: 'layout',
+    docs: {
+      description: 'There should be a break after parenthesis',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: null,
+    schema: [],
+    messages: {
+      expectedAfter: 'Expected newline after \'(\'.'
+    }
+  },
+
+  create: function(context) {
+    const sourceCode = context.getSourceCode();
+    const lines = sourceCode.lines;
+
+    return {
+      Program: function checkParenSpaces(node) {
```

Comment:
Do we need the function name there?

Reply:
Why? We use anonymous function everywhere in the codebase, why it's failing just here?

Reply:
Oh, sorry I misunderstood your comment, I was thinking that you want me to remove `function`. Fixed now.